config_cam_baseline_example.yaml

```diff
@@ -79,6 +79,15 @@ diag_basic_info:
     #Location where diagnostic plots are stored:
     cam_diag_plot_loc: /some/where/you/want/to/have/diag_plots #MUST EDIT!
 
+    #Use default variable plot settings?
+    #If "true", then variable-specific plotting attributes as defined in
+    #ADF/lib/adf_variable_defaults.yaml will be used:
+    use_defaults: false
```

Comment:
Why would we not set this to `true` going forward? Should be used where variable defaults exist.